index.css

```diff
@@ -0,0 +1,661 @@
+  justify-content: center;
+  align-items: center;
+  font-weight: bold;
+  text-align: center;
+  padding: 40px 0 40px 0;
+}
+
+.under-line {
+  align-items: center;
+  font-family: "lato", sans-serif;
+  font-size: 24px;
+  color: #272a31;
+  font-weight: bold;
+  text-align: center;
+  padding-bottom: 5px;
+}
+
+hr.line-speaker {
+  width: 80px;
+  border: 1px solid red;
+}
+
+.trainers {
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  margin-top: 50px;
+}
+
+.profesionalist {
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 40px;
+}
+
+.right {
+  margin-right: 10px;
+}
+
+.right > img {
+  width: 150px;
+  height: auto;
+}
+
+.left {
+  display: flex;
+  flex-direction: column;
+}
+
+.left > h2 {
+  font-family: "lato", sans-serif;
+  font-size: 24px;
+  color: #272a31;
+  text-align: left;
+  padding-bottom: 10px;
+  padding-left: 20px;
+}
+
+.title {
+  font-family: "lato", sans-serif;
+  font-size: 18px;
+  color: #ec5242;
+  text-align: left;
+  padding-bottom: 20px;
+  padding-left: 20px;
+}
+
+.resume {
+  font-family: "lato", sans-serif;
+  font-size: 14px;
+  color: #272a31;
+  padding-bottom: 10px;
+  padding-left: 20px;
+  text-align: left;
+}
+
+/* feature spreaker end */
+.some-right {
+  display: none;
+}
+
+@media only screen and (min-width: 768px) {
+  .humberger {
+    display: none;
+  }
+
+  .main-page {
+    width: 100vw;
+  }
+
+  .first-page {
+    width: 100%;
+    background-image: url(/assets/background-desktop.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    margin: 0;
+  }
+
+  .toolbar-logout {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .logout {
+    display: flex;
+    background-color: #545454;
+    justify-content: end;
+    padding: 20px;
+    padding-right: 40px;
+  }
+
+  .logout > a {
+    font-family: "lato", sans-serif;
+    font-size: 16px;
+    color: #d3d3d3;
+    text-decoration: none;
+    padding-right: 30px;
+  }
+
+  .fb {
+    font-size: 20px;
+    color: #fff;
+    padding-right: 10px;
+  }
+
+  .twit {
+    color: #fff;
+    font-size: 20px;
+  }
+
+  .toolbar {
+    display: flex;
+    background-color: #fff;
+  }
+
+  .image-a {
+    width: 30%;
+    padding-left: 70px;
+  }
+
+  .image-a > img {
+    padding: 10px 0 10px;
+    padding-left: 40px;
+    width: 87px;
+  }
+
+  .toolbar > ul {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    width: 65%;
+    list-style: none;
+  }
+
+  .toolbar > ul > li {
+    font-family: "lato", sans-serif;
+    font-size: 16px;
+    color: #272a31;
+  }
+
+  .agenda {
+    display: flex;
+    align-items: center;
+    padding: 100px;
+    width: 100%;
+    height: 100vh;
+  }
+
+  .hello {
+    font-size: 40px;
+  }
+
+  .main-topic {
+    font-size: 56px;
+  }
+
+  .introductory {
+    font-family: "lato", sans-serif;
+    font-size: 20px;
+  }
+
+  .date {
+    font-size: 32px;
+    color: #272a31;
+    font-weight: bold;
+  }
```

Comment:
- [ ] Please make sure that your HTML elements are aligned in the same way as the Figma design because it is great to have the skills to create pixel matching design and it's in the project requirement:

## Your design:

![hom](https://user-images.githubusercontent.com/95412943/227713052-c88ea523-90b1-460f-a65d-a5fadf4e5c12.PNG)

## Figma design:

![homfig](https://user-images.githubusercontent.com/95412943/227713063-80ab12aa-a9e7-4b4f-b792-943fcf35aef5.PNG)